### PR TITLE
KIN-209: Edit measurement solution view

### PR DESCRIPTION
**Pull request by CarlosJCD**
[KIN-209](https://loopcrack.atlassian.net/browse/KIN-209) Small [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=loopcrack-org&utm_campaign=badge_info)

### Problem

The measurement solutions entities content must be changed through a visual way

### Solution

An "edit measurement" solution form was implemented to edit existing measurement solutions records

### Requirements checklist

Include a list of requirements necessary to consider this pull request complete

- [x] It should be editable through the edit category button found in the category table.

- [x] It must load the following data loaded from the database:
    - Measurement solution name
    - Related icon
    - Background image
    - Informative text

### Testing

- [x] Go to the measurement solutions view of the admin panel and click on the edit button.
- [x] On the edit view, you must see the measurement solution content (text only) loaded on it input.
- [x] For testing the response, you need to fill the files inputs. For this in case of not having an .svg file and an image file for the inputs, you can use the ones located on `public/assets/images/` so that you can submit the form and see the response.
- [x] Finally, when submitting the form, this must return the user to the  measurement solutions page with a modal alert indicating success.

## Demo
[Demo Video](https://drive.google.com/file/d/1g_Rt4pgVTyjEn5qXdKp-xzmopkheDvdP/view?usp=sharing)

[KIN-209]: https://loopcrack.atlassian.net/browse/KIN-209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by CarlosMay7 — APPROVED** [review]

File: app/Views/admin/typeSolutions/Solutions.php [hunk 1]
```diff
@@ -47,66 +47,23 @@
...
-                                    <td>1</td>
+                                    <td><?= $measurementSolution['msId'] ?></td>
                                     <td>
                                         <div class="d-flex align-items-center fw-medium">
-                                            <p>Medidores Residenciales</p>
+                                            <p><?= $measurementSolution['msName'] ?></p>
                                         </div>
                                     </td>
-                                    <td>Icono.jpeg</td>
+                                    <td> <img class="mw-100 object-fit-contain" src="<?= $measurementSolution['fileRoute'] ?>" alt="Measurement Solution Icon"></td>
```
Comment (LUGAMAFE) [inline app/Views/admin/typeSolutions/Solutions.php:58]:
```suggestion
```
To be honest i dont like the idea of showing the icons in the table can we remove this column please

**Review by LUGAMAFE — APPROVED** [review]
Nice @CarlosJCD just please remove the icon column preview from the table

**Review by Cristianpan — APPROVED** [review]

**Review by Oswaldo-Chan — APPROVED** [review]
Good job, @CarlosJCD! Just keep in mind to make the changes that @LUGAMAFE  mentioned

**Review by CheloChidoone — APPROVED** [review]
Good job @CarlosJCD, thanks for making the changes Luis asked for.

**Pull request merged**